neo.UnitTests/Network/RPC/Models/UT_RpcNep5Balances.cs

```diff
@@ -0,0 +1,60 @@
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.IO.Json;
+using Neo.Network.RPC.Models;
+using System.Numerics;
+
+namespace Neo.UnitTests.Network.RPC.Models
+{
+    [TestClass]
+    public class UT_RpcNep5Balances
+    {
+        private RpcNep5Balances balances;
+
+        [TestInitialize]
+        public void Setup()
+        {
+            balances = new RpcNep5Balances()
+            {
+                Address = "abc",
+                Balances = new RpcNep5Balance[] {
+                    new RpcNep5Balance()
```

Comment:
Maybe add two RpcNepBalance?

Reply:
@eryeer, you marked as resolved.

I mean, should this test the behavior with, at least, two balances? Or one is enough?